src/App.js

```diff
@@ -12,17 +10,13 @@ import ProductList from './components/views/ProductList/ProductList';
 import ProductPage from './components/views/ProductPage/ProductPage';
 
 const App = () => (
-  <Provider store={store}>
-    <BrowserRouter>
-      <MainLayout>
-        <Switch>
-          <Route exact path={'/'} component={Homepage} />
-          <Route exact path={'/shop/:categoryId'} component={ProductList} />
-          <Route exact path={'/product/:productId'} component={ProductPage} />
-        </Switch>
-      </MainLayout>
-    </BrowserRouter>
-  </Provider>
+  <MainLayout>
```

Comment:
Dlaczego provider został usunięty/przeniesiony?

Reply:
Rozmawialiśmy o tym na kanale, że był problem z testem i w pierwszym poście mi napisałeś, żeby zrobić według tego co mi się udało wygooglować https://chat.kodilla.com/group/project-wdp-2206-02?msg=6iQFDf6DYawoDt2CW 

Na koniec naszej konwersacji wyszło, że trzeba było dodać provider do pliku z testem. Natomiast podczas szukania informacji na ten temat było sporo wątków mówiących o tym, że tego typu błąd się często pojawia, jeżeli komponent App nie jest objęty w Provider i tak też robiliśmy podczas naszego kursu. Jeżeli Provider ma wrócić do App.js to daj znać i poprawię :).

Reply:
Na początku nie zrozumiałem dokładnie z czym masz problem, ale później doszliśmy do tego że po prostu trzeba dodać provider do testów, a tutaj nie trzeba go ruszać

Reply:
Poprawione